model/SearchEngine/Driver/Elasticsearch/QueryBuilder.php

```diff
@@ -174,6 +175,27 @@ public function getSearchParams(
         return $params;
     }
 
+    public function getBoolQuery(string $field, array $values, ?int $size = self::DEFAULT_SIZE): array
+    {
+        return [
+            'index' => 'tests',
```

Comment:
It's a `getBoolQuery` not a `getBoolQueryForTests` overall not the best idea to limit query builder to a specific index